front/src/modules/simulationResult/components/ChartHelpers/enableInteractivity.tsx

```diff
@@ -264,16 +290,24 @@ export const enableInteractivity = <
 ) => {
   if (!chart) return;
   const zoom = d3zoom<SVGGElement, unknown>()
-    .scaleExtent([0.3, 20]) // This controls how much you can unzoom (x0.5) and zoom (x20)
+    .scaleExtent([0.3, 20]) // This controls how much you can unzoom (x0.3) and zoom (x20)
     .extent([
       [0, 0],
       [chart.width, chart.height],
     ])
     .wheelDelta(wheelDelta)
+    // Allows evenements to be triggered on zoom and drag interactions for all graphs
     .on('zoom', (event) => {
       event.sourceEvent.preventDefault();
       const zoomFunctions = updateChart(chart, keyValues, additionalValues, rotate, event);
-      const newChart = { ...chart, x: zoomFunctions.newX, y: zoomFunctions.newY };
+      // Overide axis with new ones from updated chart
+      const newChart = {
+        ...chart,
+        x: zoomFunctions.newX,
```

Comment:
Do you think you can rename `zoomFunctions` ? I think the name is not very appropriate, maybe newAxis would be better ? zoomedAxis ? updatedAxis ?